### ISSUE TITLE
Breakpad on OSX always fail to initialise

### ISSUE THREAD

**Issue by nrako** [nwjs/nw.js#2518]
I've never been able to have the crash dump working, and I always have the following log

`[60832:1027/110731:ERROR:breakpad_mac.mm(238)] Breakpad initializaiton failed`

Which is emitted around here https://github.com/rogerwang/node-webkit/blob/0afb5476f5cbd29d7efa299acedc28d86ae7cdd9/src/breakpad_mac.mm#L236-L240

Right now I have a very odd crash with 0.11.0-rc1 which I ain't able to isolate, having a crash dump might help.

**Comment by adam-rocska-vs** [comment]
Hi.
May I kindly ask what is the resolution status of this rather rusty issue?

**Comment by rogerwang** [comment]
this was an upstream issue and will try to merge the wordaround in the next version:

https://code.google.com/p/chromium/issues/detail?id=414239
https://code.google.com/p/google-breakpad/issues/detail?id=612

**Comment by adam-rocska-vs** [comment]
@rogerwang Thank you very much for your quick response.

**Comment by brzpegasus** [comment]
@rogerwang I see you released a new version two days ago, but I'm still getting the breakpad init error. I'm guessing the workaround you mentioned will be for 0.11.7 then?

**Comment by zwhitchcox** [comment]
I'm getting the same error

**Comment by creatorkuang** [comment]
I have the same problem, but when i run it with "sudo", it work.

**Comment by sorrycc** [comment]
Same problem with me, and `sudo` don't work.

**Comment by marnen** [comment]
Having the same issue (I haven't tried `sudo` yet). The "hello world" application exhibits this error.

**Comment by rogerwang** [comment]
fixed in #3226

**Comment by GantMan** [comment]
Hi!  New to NWJS.

My experience:  `npm install nodewebkit -g` :+1: 
`nodewebkit`
```
[16747:1111/080657:ERROR:breakpad_mac.mm(238)] Breakpad initializaiton failed
[16748:1111/080657:ERROR:breakpad_mac.mm(238)] Breakpad initializaiton failed
[16749:1111/080719:ERROR:breakpad_mac.mm(238)] Breakpad initializaiton failed
[16750:1111/080724:ERROR:breakpad_mac.mm(238)] Breakpad initializaiton failed
```
:-1:
